static/js/settings_users.js

```diff
@@ -114,14 +114,14 @@ function populate_users(realm_people_data) {
 
     var bots_table_html = "";
     _.each(bots, function (user) {
-        var bot_html = templates.render("admin_user_list", {user: user});
+        var bot_html = templates.render("admin_user_list", {user: user, is_loggedin_user_admin: page_params.is_admin});
         bots_table_html = bots_table_html.concat(bot_html);
     });
     bots_table.append(bots_table_html);
 
     _.each(active_users, function (user) {
         var activity_rendered;
-        var row = $(templates.render("admin_user_list", {user: user}));
+        var row = $(templates.render("admin_user_list", {user: user, is_loggedin_user_admin: page_params.is_admin}));
```

Comment:
Oh, ok, I understand now.  The name `is_loggedin_user_admin` should be spelled `is_logged_in_user_admin` (note the underscore), but I think we can come up with a better name.  I would suggest `can_modify` or `read_only` to indicate the powers of the logged in user.